src/components/Breadcrumbs/index.tsx

```diff
@@ -16,11 +16,14 @@ type Props = {
 };
 
 const overrides = {
-  '/': 'Home'
+  '/': 'Home',
+  '/[platform]/how-amplify-works': 'How Amplify works'
```

Comment:
Do we need these changes to Breadcrumbs if we add the page to directory.mjs? I'm not sure if there is a reason we need to treat this page as special in the menu or if it can just work like the others (@timngyn?)

Reply:
Oh yes I didn't notice this was not added to directory.mjs, sorry about that. I don't think we should have to change breadcrumbs if this gets added there

Reply:
Ah Just talked to Katie about it, I think right now the "How Amplify Works" text is hard coded in the Menu as a prop: https://github.com/aws-amplify/docs/blob/36e47a8589fdd34fb22ed0ebba0c974533223e55/src/components/Layout/Layout.tsx#L274-L290

I think if Katie adds it to directory.mjs then it gets shown twice. So this might require additional changes. Is this blocking? If it is I think I can try and make those updates @hbuchel